arch/arm/core/cpu_idle.S

```diff
@@ -21,19 +21,17 @@
 _ASM_FILE_PROLOGUE
 
 GTEXT(z_CpuIdleInit)
-#ifdef CONFIG_SYS_POWER_MANAGEMENT
```

Comment:
Could you please, remind me why this has been cleaned-up?

Reply:
OH, I see, it is arch: arm: cpu_idle: Remove unused functions 
Could you split that out in a standalone PR? That's a clean up patch - not belonging here.

Reply:
..and that PR can be reviewed very fast, of course.

Reply:
Submitted #18099